packages/openid4vc/src/openid4vc-verifier/repository/SphereonOpenId4VcRelyingPartyEventEmitter.ts

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Is this needed? Might be good to just add them above the uses so we do not forget to fix them later. On top of the file i tend to miss.